packages/cli/lib/commands/build.js

```diff
@@ -94,15 +93,6 @@ async function command(src, argv) {
 	argv.production = toBool(argv.production);
 
 	let cwd = resolve(argv.cwd);
-	let modules = resolve(cwd, 'node_modules');
-
-	if (!isDir(modules)) {
-		return error(
-			'No `node_modules` found! Please run `npm install` before continuing.',
-			1
-		);
-	}
-
```

Comment:
If you're in a monorepo and the hoisting is good this folder wont exist so checking if the project is installed like this is not correct